data/core/encyclopedia/geography.cfg

```diff
@@ -201,26 +209,29 @@ Yet despite the sheer wilderness above the Northlands, the subterranean Dwarvish
     id=far_north
     title= _ "Far North"
     text= _ "Cold, harsh, and inaccessible, the Far North is the ancestral home of the Orcish Clannate. It lies north of the <ref>dst='heart_mountains' text='Heart Mountains'</ref>, which the Orcs call the Haggid-Dargor and claim (without merit) as their own. To the east lie the Unaligned Tribes of the Wild Steppe, who fell out of the control of the Clannate, instead roaming with wild human barbarians and clashing with the High Elves of the North Plains (known as North Elves in human lands). The High Elves themselves reside further east, where it is rumored they rule a vast kingdom." + "
+
 " +
         #po: Geography of the Far North
         _ "<header>text='Notable cities:'</header>
     • Barag Gor: A city home to the Orcish Council.
     • Bitok
     • Borstep
     • Farzi
-    • Jotha: A large fortified capital of the local merfolk that lies within the bay of Jotha.
+    • Jotha: A large fortified capital of the local merfolk that lies within the Bay of Jotha.
```

Comment:
Question: Is Bay of Jotha actually a noteworthy location deserving of its own name? Also just realised this (existing) description is self-referential. A capital lying within the bay of the capital?

Reply:
I replaced this with “South of Bitok”.